src/static/riot/competitions/detail/_tabs.tag

```diff
@@ -246,47 +173,23 @@
                 self.competition.is_admin = CODALAB.state.user.has_competition_admin_privileges(competition)
                 self.update()
                 $('.tabular.menu .item').tab();
+                _.forEach(competition.pages, (page, index) => {
+                    $(`#page_${index}`)[0].innerHTML = sanitize_HTML(EasyMDE.prototype.markdown(page.content))
+                })
             })
-            //self.competition = opts.competition
-
-            // Handling tabs
-            self.tabs = ["learn_details", "phases", "participate", "results"]
-            self.active_tab = self.tabs[0]
-
-            // Dynamic based on which `tab` object is active
-            self.active_tab_active_subtab = {}
-            self.active_tab_subtabs = {}
-
-            // handling pages if we're on learn the details
-            if (self.competition.pages) {
-                self.active_page = self.competition.pages[0]
-            }
 
             self.one("mount", function () {
-                // tabs
-                $('.tabular.menu .item').tab(); // Activate tabs
-                // $('.menu .item').tab()
-            })
 
-            self.set_active_tab = function (index) {
-                // On set active, we need to remove the current active_tab's `active` class, and add it to the new tab
-                self.active_tab = self.tabs[index]
-            }
+            })
 
-            self.tab_string_to_index = function (string_tab_id) {
-                for (i = 0; i < self.tabs.length; i++) {
-                    if (self.tabs[i] === string_tab_id) {
-                        return i
-                    }
-                }
-            }
-            self.toggle_competition_publish = function (competition) {
-                CODALAB.api.toggle_competition_publish(competition.id)
+            self.toggle_competition_publish = function () {
+                let comp_id = self.competition.id
+                CODALAB.api.toggle_competition_publish(comp_id)
                     .done(function (data) {
                         var published_state = data.published ? "published" : "unpublished"
                         toastr.success(`Competition has been ${published_state} successfully`)
                         self.update()
-                        CODALAB.api.get_competition(self.competition.id)
+                        CODALAB.api.get_competition(comp_id)
```

Comment:
hmm, i think the `comp_id` temp holder is kinda gross here? would prefer `self.competition.id`?